docs/2101UserGuide.md

```diff
@@ -109,110 +109,106 @@ Examples:
-* At least one field to edit must be provided.
-* `EVENT_NAME` **must only contain** Alphanumeric Characters and spaces, and it should not be blank
-* `LOCATION` **must not** be blank and can take in any values.
-* `DATE` **must be a valid date** in the format of **YYYY-MM-DD**.
-* `TAG` **must only contain** Alphanumeric Characters with no space in between.
+Here is an example command where we delete the event at **index** 1:<br>
+`deleteEvent 1`
 
-Examples:
-* `editEvent 5 n/Halloween Surprise Party l/UTR d/2023-10-31 t/sem1` edits the 5th event in the event list to change the name to `Halloween
-  Surprise Party`, the location to `UTR`, the date to `2023-10-31` and the tag to `sem1`.
-* `editEvent 3 l/UCC Theater` edits the 3rd event in the event list to change the location to `UCC Theater`.
+<div markdown="block" class="alert alert-info">:information_source: The `EVENT_INDEX` parameter refers to the index number shown in the **currently displayed** event list.
+</div>
 
 <hr class="feature-class-separator">
 
-### Enrolment commands
+### Enrolment Commands
 
-#### Enrol a Member to an Event: `enrol`
+Enrolments are a way for you to manage your members’ involvement in your events. The enrolment commands below are the bread-and-butter for your participation tracking needs:
 
-Enrols a member to an event.
+#### Enrol a Member to an Event: `enrol`
 
-Format: `enrol m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]`
+Now that you have added all your members and all your events, keeping track of your members’ attendance to any event is as simple as using the `enrol` command!
 
-* Enrols the member at the specified `MEMBER_INDEX` to the event at the specified `EVENT_INDEX` with `NUMBER_OF_HOURS` specifying the number of hours that the member contributed and `REMARK` stating extra remarks about the member and event.
-* The `MEMBER_INDEX`/`EVENT_INDEX` refers to the index number shown in the **currently displayed** member/event list.
-* The `MEMBER_INDEX`/`EVENT_INDEX` **must be a positive integer** that is within the range of the length of the member/event list.
-* The `NUMBER_OF_HOURS` **must be a positive integer** and **must be less than or equal to 2147483647**.
+Here’s how you can use it:<br>
+`enrol m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]`
 
-Examples:
-* `enrol m/1 e/5 h/3 r/did planning` enrols the 1st member in the member list to the 5th event in the event list, where the member had 3 hours of contributions to that event and has a remark stating that the member "did planning".
-* `enrol m/5 e/1` enrols the 5th member in the member list to the 1st event in the event list.
+Here is an example command where we enrol the member at **member index** 1 to the event at **event index** 1, where our member
+has contributed 2 **hours**, and we want to make a "Role: Photographer" **remark** to reflect their role as a photographer:<br>
```

Comment:
Do you think this will be better:
```suggestion
Here is an example command where we enrol the member at **member index** 1 to the event at **event index** 1, where our member
has contributed 2 **hours**, and we want to leave a **remark** "Role: Photographer" to reflect their role as a photographer:<br>
```

Reply:
Done!